src/App.tsx

```diff
@@ -1,48 +1,67 @@
+import React from 'react';
+import {
+  NavLink,
+  Navigate,
+  Routes,
+  Route,
+} from 'react-router-dom';
+
+import classNames from 'classnames';
+
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import { TabsPage } from './TabsPage';
 
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+export const App: React.FC = () => {
+  const getLinkClass = ({ isActive }: { isActive: boolean }): string => {
+    return isActive ? 'is-active' : '';
+  };
 
-const App = () => (
-  <>
-    {/* Also requires <html class="has-navbar-fixed-top"> */}
-    <nav className="navbar is-fixed-top has-background-light" data-cy="nav">
-      <div className="navbar-menu">
-        <div className="navbar-start">
-          <a href="/" className="navbar-item isActive">Home</a>
-          <a href="/tabs" className="navbar-item isActive">Tabs</a>
-        </div>
-      </div>
-    </nav>
+  return (
+    <div>
+      <section
+        className="
+          navbar
+          navbar-start
+          is-fixed-top
+          has-background-light
+        "
+        data-cy="nav"
+      >
+        <article className="navbar-menu">
+          <NavLink
+            to="/"
+            className={param => classNames(
+              'navbar-item',
+              getLinkClass(param),
+            )}
```

Comment:
I assume you could implement something like this one
```
const getClassNames = ({ isActive }) => {
  ...
};

...
<NavLink
  ...
  className={getClassNames}
```

Reply:
like, you could return `navbar-item` from `getLinkClass` and avoid using `classnames`